### PR TITLE
Add tests for validate_pr workflow

### PR DESCRIPTION
**Pull request by landon-martin**
Some validation of the validate_pr workflow while developing it to set expectations. Also improved logging when the validation passes and fails.

Example error logging: 
```
PR Title does not match regex pattern!
Please follow format: "<JIRA_PROJ>-<JIRA_NUM>: <TITLE>"
Example: "JIRA-404: Title Here"
```

### CODE REVIEW

File: .github/workflows/validate_pr_test.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/validate_pr.yml
+      - .github/workflows/validate_pr_test.yml
+
+jobs:
+  with_space:
+    name: With Space - Expect Pass
```
Comment (ja2nicholl) [inline .github/workflows/validate_pr_test.yml:11]: Can you add a "Revert" example as well for a test?

**Review by ja2nicholl — COMMENTED** [review]
Looks great to have some tests, just minor comment.

File: .github/workflows/validate_pr_test.yml (hunk 1 above)
Comment (landon-martin) [reply .github/workflows/validate_pr_test.yml:11]: Oh, definitely! One sec.

Comment (landon-martin) [reply .github/workflows/validate_pr_test.yml:11]: Done.

**Review by ja2nicholl — APPROVED** [review]

**Pull request merged**